src/main/java/com/microsoft/aad/msal4j/AadInstanceDiscoveryProvider.java

```diff
@@ -205,7 +205,7 @@ else if (autoDetect) {
 
     private static String formInstanceDiscoveryParameters(URL authorityUrl) {
         return INSTANCE_DISCOVERY_REQUEST_PARAMETERS_TEMPLATE.replace("{authorizeEndpoint}",
-                getAuthorizeEndpoint(authorityUrl.getAuthority(),
+                getAuthorizeEndpoint(authorityUrl.getHost(),
```

Comment:
if port specified for authority, it probably should be used with derived endpoints like authorize and instance discovery, is not it ?

Reply:
Can you give me more details on what you mean by that? Do you think the fix is not complete and I need to make changes in other areas as well?

Reply:
SomkaPe and I discussed these changes offline. This change is not actually changing the authorization endpoint, instead its setting the authorization endpoint query parameter in the instance discovery URL. No changes needed on your end, I'll take care of finishing touches before merging.